mscore/pianoroll/pianoview.cpp

```diff
@@ -1349,16 +1371,16 @@ void PianoView::changeChordLength(const QPointF& pos) {
 //   roundToStartBeat
 //---------------------------------------------------------
 
-Fraction PianoView::roundToStartBeat(int tick, bool down)  const
+Fraction PianoView::roundToNearestBeat(int tick, bool down)  const
       {
       Score* _score = _staff->score();
       Pos barPos(_score->tempomap(), _score->sigmap(), tick, TType::TICKS);
-
-      int beatsInBar = barPos.timesig().timesig().numerator();
+      
```

Comment:
whitespace damage

Reply:
What's wrong here?  Is it still in the rebase?

Reply:
Not in this PR anymore, but still in your code, from your last PR